src/PrestaShopBundle/Form/DataTransformer/ArabicToLatinDigitDataTransformer.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\DataTransformer;
+
+use PrestaShop\PrestaShop\Core\Util\ArabicToLatinDigitConverter;
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * Class ArabicToLatinDigitDataTransformer is responsible for converting arabic/persian digits to latin digits
+ */
+final class ArabicToLatinDigitDataTransformer implements DataTransformerInterface
+{
+    /**
+     * @var ArabicToLatinDigitConverter
+     */
+    private $arabicToLatinDigitConverter;
+
+    public function __construct(ArabicToLatinDigitConverter $arabicToLatinDigitConverter)
+    {
+        $this->arabicToLatinDigitConverter = $arabicToLatinDigitConverter;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function transform($value)
+    {
+        return $value;
```

Comment:
I the dual transformation is managed by doing nothing here (I see you don't call `$this->arabicToLatinDigitConverter->reverseConvert`) Then this needs to be thoroughly documented and unit tested to avoid anybody changing this behaviour